src/hooks/useContract.ts

```
@@ -67,12 +70,15 @@ const useContract = <T extends ExecutionType, R = any>(
-              functionName,
-              args,
-            });
+            const { result }: { result: bigint } = await simulateContract(
+              config as any,
+              {
+                abi,
+                address: facet === "ERC20" ? contractAddress : DIAMOND_ADDRESS,
+                functionName,
+                args,
+              }
+            );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)